### DISCUSSION TITLE
Dark theme all window title bars on Windows 10

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#178] [Feature requests] [answered]
On Windows, the title bar is white. If this could be updated to a dark theme, then the Live Stats window would match the contents of the window.

**Comment by f1multiviewer** [comment, marked as answer]
Will be fixed in the next release, already in the latest beta (image by SkyWeb):

![image](https://user-images.githubusercontent.com/106821994/196927272-769c35f1-3d44-4aac-a687-c8666108d391.png)

(if you want to try the beta: https://api.f1mv.com/api/v1/releases/latest/windows/download?channel=beta)
